pom.xml

```diff
@@ -21,14 +21,33 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.canelmas.kafka</groupId>
-    <artifactId>connect-s3-fieldandtime-partitioner</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <artifactId>connect-fieldandtime-partitioner</artifactId>
+    <version>stuart-1.0.0</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
     <packaging>jar</packaging>
 
     <properties>
-        <confluent.version>5.3.0-SNAPSHOT</confluent.version>
+        <confluent.version>5.4.1</confluent.version>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>confluent</id>
+            <url>https://packages.confluent.io/maven/</url>
```

Comment:
We don't have the deps in nexus? I think I just saw 5.4.0 not 5.4.1, idk if we could use that...

Reply:
You're right! I'll change to 5.4.0 because this is what we were using in the previous plugin! And instead of adding the confluent repository I'll add the nexus one! wdyt?

Reply:
That'd be great! Thanks!